backend/src/schema/resolvers/notifications.spec.js

```diff
@@ -1,26 +1,34 @@
 import { GraphQLClient } from 'graphql-request'
 import Factory from '../../seed/factories'
 import { host, login, gql } from '../../jest/helpers'
+import { neode } from '../../bootstrap/neo4j'
 
-const factory = Factory()
 let client
+const factory = Factory()
+const instance = neode()
 const userParams = {
   id: 'you',
   email: 'test@example.org',
   password: '1234',
 }
+const categoryIds = ['cat9']
 
 beforeEach(async () => {
   await factory.create('User', userParams)
+  await instance.create('Category', {
+    id: 'cat9',
+    name: 'Democracy & Politics',
+    icon: 'university',
+  })
 })
 
 afterEach(async () => {
   await factory.cleanDatabase()
 })
 
-describe('query for notification', () => {
+describe('Notification', () => {
```

Comment:
this was an unintended change @Tirokk when dealing with a merge conflict, I can revert it, if you want